.travis.yml

```diff
@@ -1,12 +1,8 @@
 language: php
 
 php:
-  - 5.3
```

Comment:
@core23 Maybe today, but maybe not tomorrow. On your case, you lock a version of symfony and don't test with a free of modification `composer.json` file.